lib/src/page_list_viewport_gestures.dart

```diff
@@ -288,35 +305,190 @@ class _PageListViewportGesturesState extends State<PageListViewportGestures> wit
+/// Velocities are slow, normal, and fast.
+/// These categories are used to individually define ballistic simulation behavior
+/// across a variety of scrolling situations.
+class GestureThresholdsAndScales {
+  /// Definition for a tiny distance in pixels.
+  ///
+  /// {@template distance_definitions}
+  /// Gesture translation distance categorization.
+  /// The launch velocity for the ballistic simulation can be individually
+  /// scaled for gestures categorized into these categories.
+  /// Distances scale diagram:
+  /// (0 ... "tiny" ... tinyDistanceMax] (... "small" ... SmallDistanceMax]( ... "large" ...
+  /// {@endtemplate}
+  static const double tinyDistanceMax = 3;
+
+  /// Definition for a small distance in pixels.
+  ///
+  /// {@macro distance_definitions}
+  static const double smallDistanceMax = 120.0;
+
+  /// Definition for a slow velocity in pixels per second.
+  ///
+  /// {@template velocity_definitions}
+  /// Gesture velocity categorization.
+  /// The launch velocity for the ballistic simulation can be individually
+  /// scaled for gestures categorized into these categories.
+  /// Velocities scale categorization diagram:
+  /// (0 ... "slow" ... slowVelocityMax] (... "normal" ... normalVelocityMax]( ... "fast" ...
+  /// {@endtemplate}
+  static const double slowVelocityMax = 300.0;
+
+  /// Definition for a normal velocity in pixels per second.
+  ///
+  /// {@macro velocity_definitions}
+  static const double normalVelocityMax = 850.0;
+
+  /// Minimal neccessary velocity when the user releases from any panning motion
+  /// required for which a ballistic simulation to be launched.
+  ///
+  /// Value is in pixels per second.
+  static const double minSmallTranslationBallisticActivationVelocity = 120.0;
+
+  /// {@template velocity_increase}
+  /// Ballistic simulation launch velocity multiplier according to the
+  /// category into which its translation distanca and reported velocity
+  /// fall.
+  ///
+  /// Used to speed up or slow down the simulation speed for different gesture kidns.
+  /// Is applied when the user releases an arbitrary direction panning motion (not locked axis), and the content goes
+  /// ballistic.
+  /// This value is unit-less and should be multiplied by a velocity that's measured in pixels
+  /// per second.
+  /// {@endtemplate}
+  static const double smallTranslationSlowVelocityMultiplier = 0.5;
+
+  /// {@macro velocity_increase}
+  static const double smallTranslationNormalVelocityMultiplier = 0.6;
+
+  /// {@macro velocity_increase}
+  static const double smallTranslationFastVelocityMultiplier = 0.7;
 
-  /// The minimum distance the viewport must be translated such that the
-  /// gesture is considered a viewport re-adjustment which results in the
-  /// momentum simulation to be aborted.
-  static double kViewportReAdjustmentMinTranslationDistance = 60;
+  /// {@macro velocity_increase}
+  static const double largeTranslationNormalVelocityMultiplier = 0.85;
```

Comment:
Why doesn't "large translation" have values for "slow" and "fast"? Also, why aren't there any tiny values?

Reply:
Didn't find any use in differentiating large translations as slow and fast. If the translation is large, we almost always want the physics simulation. We are not launching the simulation for tiny translations, so there is no associated velocity.